src/Command/RecipesCommand.php

```diff
@@ -0,0 +1,253 @@
+            $this->displayPackageInformation(current($recipes));
+
+            return 0;
+        }
+
+        // display a resume of all packages
+        $write = [
+            '',
+            '<bg=blue;fg=white>                      </>',
+            '<bg=blue;fg=white> Recipes available.   </>',
+            '<bg=blue;fg=white>                      </>',
+            '',
+        ];
+
+        /** @var Recipe $recipe */
+        foreach ($recipes as $name => $recipe) {
+            $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
+
+            $additional = '';
+            if (null === $lockRef && null !== $recipe->getRef()) {
+                $additional = '<comment>(recipe not installed)</comment>';
+            } elseif ($recipe->getRef() !== $lockRef) {
+                $additional = '<comment>(update available)</comment>';
+            }
+            $write[] = sprintf(' * %s %s', $name, $additional);
+        }
+
+        // TODO : Uncomment the lines following the implemented features
+        $write[] = '';
+        $write[] = '<fg=blue>Run</>:';
+        $write[] = ' * composer symfony:recipes vendor/package to see details about a recipe.';
+        //$write[] = ' * composer symfony:recipes:update vendor/package to update that recipe.';
+        //$write[] = ' * composer symfony:recipes:blame to see a tree of all of the installed/updated files.';
+        $write[] = '';
+
+        $this->getIO()->write($write);
+
+        return 0;
+    }
+
+    private function displayPackageInformation(Recipe $recipe)
+    {
+        $recipeLock = $this->symfonyLock->get($recipe->getName());
+
+        $lockRef = $recipeLock['recipe']['ref'] ?? null;
+        $lockFiles = $recipeLock['files'] ?? null;
+
+        $status = '<comment>up to date</comment>';
+        if ($recipe->isAuto()) {
+            $status = '<comment>auto-generated recipe</comment>';
+        } elseif (null === $lockRef && null !== $recipe->getRef()) {
+            $status = '<comment>recipe not installed</comment>';
+        } elseif ($recipe->getRef() !== $lockRef) {
+            $status = '<comment>upate available</comment>';
+        }
+
+        $io = $this->getIO();
+        $io->write('<info>name</info>     : '.$recipe->getName());
+        $io->write('<info>version</info>  : '.$recipeLock['version']);
+        $io->write('<info>repo</info>     : '.$recipeLock['recipe']['repo']);
+        $io->write('<info>sha</info>      : '.$lockRef);
+        $io->write('<info>status</info>   : '.$status);
+
+        if (null !== $lockFiles) {
+            $io->write('<info>files</info>    : ');
+            $io->write('');
+
+            $tree = $this->generateFilesTree($lockFiles);
+
+            $this->displayFilesTree($tree);
+        }
+
+        // TODO : Uncomment the line when the feature will be implemented
+        //$io->write([
+        //    '',
+        //    'Update this recipe by running composer symfony:recipe:update symfony/framework-bundle',
```

Comment:
```suggestion
        //    'Update this recipe by running composer:',
        //    '<info>symfony:recipe:update symfony/framework-bundle -v</info>'
```

That's some reformatting, but I'd also like to add the `-v` to it - that will give a bit more information about what the recipe update is doing (what files it's creating, for example).